examples/pony/alphabet/Makefile

```diff
@@ -11,7 +11,7 @@ $(abspath $(lastword $(MAKEFILE_LIST)))_MK := 1
 TEST_TARGET := false
 
 # uncomment to disable generate pony related targets (build/test/clean) for pony sources in this directory
-#PONY_TARGET := false
+PONY_TARGET := false
```

Comment:
This seems to be an error. Why are we disabling the the pony application build/test/clean targets for a pony applications?